Orange/preprocess/impute.py

```diff
@@ -129,6 +141,9 @@ def __call__(self, data, variable, *, default=None):
     def copy(self):
         return Default(self.default)
 
+    def __repr__(self):
+        return "Default()"
```

Comment:
`Default` constructor accepts arguments and this `__repr__ doesn't account for that (same with `ImputeSql` above). Could the [`Reprable` idea](https://github.com/biolab/orange3/pull/1507/files#r78161092) be used here as well?